src/Directory.Build.props

```diff
@@ -44,6 +44,7 @@
     </ProjectReference>
     <PackageReference Condition="'$(UseStyleCopAnalyzer)' != 'false' and '$(MSBuildProjectExtension)' == '.csproj'"
       Include="StyleCop.Analyzers" Version="1.1.0-beta008" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.0-beta2.final" PrivateAssets="All" />
```

Comment:
This adds the FxCop Analyzer on all projects. When working on fixing its errors it might be desirable to only add the analyzer on specific projects instead of all of them. This can be done by adding the `<PackageReference>` only on the .csproj of the project we're interested on having the analyzer.